src/js/i18n/ta.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Created by Tim on 2/1/14.
+ */
+(function () {
+  angular.module('ui.grid').config(['$provide', function($provide) {
+    $provide.decorator('i18nService', ['$delegate', function($delegate) {
+      $delegate.add('en', {
```

Comment:
This is the only line that needs to be fixed. Please change this to `$delegate.add('ta', {` and we can merge this in.